extensions/wikia/PageHeader/styles/index.scss

```diff
@@ -59,8 +59,16 @@
 		padding-bottom: 12px;
 
 		.wds-dropdown__toggle-chevron {
+			margin-left: 4px;
+			transform: scale(.75);
```

Comment:
why such style? shouldn't we use just smaller icon?

Reply:
we don't have smaller icon, this is the smallest one... but it is a bit bigger than the one we have on production right now and Megan asked to make it smaller. I don't like it either, but it doesn't look good without this minimizing.

Reply:
shouldn't this be done in DS?

Reply:
I don't think so. Right now we use this xxs icon in only one place, and I think this is an exception,  which we should not copy.